test/node-query-test.js

```diff
@@ -0,0 +1,57 @@
+import assert from 'assert';
+import Client from '../src/client';
+import Config from '../src/config';
+import nodeQuery from '../src/node-query';
+
+suite('node-query-test', () => {
+  const querySplitter = /[\s,]+/;
+
+  function tokens(query) {
+    return query.split(querySplitter).filter((token) => Boolean(token));
+  }
+
+  const config = new Config({
+    domain: 'sendmecats.myshopify.com',
+    storefrontAccessToken: 'abc123'
+  });
+
+  const client = new Client(config);
+
+  test('it converts the ID to GID format', () => {
+    const productQuery = nodeQuery('Product', ['title']);
+    const query = client.graphQLClient.query((root) => {
```

Comment:
@jzjiang not sure if there's a more direct way to test this. Technically I can just pass a selection into `productQuery` but wasn't sure if this is the easiest way to do it.